src/components/movieForm/index.tsx

```diff
@@ -12,12 +12,42 @@ import useStyles from './styles';
 import { IMovie, OptionOnSelectData } from '../../shared/types';
 import { TriangleDown12Filled } from '@fluentui/react-icons';
 import { genres } from '../../shared/constants';
+import { Controller, SubmitHandler, useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import * as yup from 'yup';
+
+const schema = yup
+  .object({
+    title: yup.string().required('Title is required'),
+    vote_average: yup
+      .string()
```

Comment:
I think it should be a number, try to use `valueAsNumber: true`. Please, add min and max values to check that the value is in bounds.
<img width="760" alt="image" src="https://user-images.githubusercontent.com/81073/235854289-b1a09e1a-127c-440d-b537-d5843f64ed4b.png">